### DISCUSSION TITLE
Radius for authentication

### DISCUSSION THREAD

**Discussion by snycko** [firefly-iii/firefly-iii#6286] [General]
Hello,

I've been trying to find out if it's possible somehow run Radius on Firefly for authentication. Without success.
Any ideas ?

Thanks

**Comment by JC5** [comment]
Firefly III doesn't support Radius, unfortunately. If you have a reverse auth proxy in front of Firefly III that supports Radius, you could use that.

**Reply by snycko** [reply]
Thank you for fast reply :)
